gson/gson/src/main/java/com/google/gson/internal/sql/SqlTimestampTypeAdapter.java

```diff
@@ -49,7 +49,8 @@ private SqlTimestampTypeAdapter(TypeAdapter<Date> dateTypeAdapter) {
   @Override
     public Timestamp read(JsonReader in) throws IOException {
```

Comment:
indentation changes needed

Reply:
@weltonfelix can you address this change in 2-3 days? Otherwise, we'll have to ask someone else to do it.